v4-client-js/src/clients/composite-client.ts

```diff
@@ -1048,6 +1048,8 @@ export class CompositeClient {
       const registry = generateRegistry();
       const msgs: EncodeObject[] = [];
 
+      const isDydxUsd = params.ticker.toLowerCase() === 'dydx-usd';
```

Comment:
Question, don't we already do some state queries in the FE, could we generalize this by just querying all markets and using the market if the `ticker` matches rather than doing this for a specific case.

Reply:
yeah agree that's better but i wanted to leave generic logic to Jared when he gets back